gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
```

Comment:
This causes big downloads on CI, mind keeping this as it was?

Reply:
Ah, damn, that was definitely not intended! Will remove this ASAP.